bin/tag-release-continue.js

```diff
@@ -24,11 +25,21 @@ const callback = async options => {
 	if (options.branch.includes("promote-release")) {
 		flow = filterFlowBasedOnDevelopBranch(options, promoteContinue);
 
+		options.scripts = await utils.getScripts("promote");
+		options.command = "promote";
+		if (options.scripts[`post${options.command}`]) {
```

Comment:
I understand `options.command` needs to be set here and on line `39`, but I feel the code would be easier to read if line `30` read like...

`if (options.scripts.postpromote) {` (likewise on line `40`)

Reply:
It's funny, I changed it because I thought you might tell me it should just go back to the templated string. 😁 Ill address this